library/src/main/java/com/datatorrent/lib/math/AbstractOutput.java

```diff
@@ -38,23 +38,23 @@
    * Double type output.
    */
   @OutputPortFieldAnnotation(optional = true)
-  public final transient DefaultOutputPort<Double> doubleResult = new DefaultOutputPort<Double>();
+  public transient DefaultOutputPort<Double> doubleResult = new DefaultOutputPort<Double>();
```

Comment:
@ilganeli This won't work. The purpose of keeping port variable final is because this variable is used in Application.java to connect the ports. Once its used in addStream in application.java, the reference to this variable is kept and the same object will be used. Hence even if the object is recreated in child class, that won't take effect.

Suggested approach for adding unifier to super class's port is as follows:

1. Do following in parent class:
``
public final transient DefaultOutputPort<Double> doubleResult = new DefaultOutputPort<>()
{
   @Override
  public Operator.Unifier getUnifier()
  {
     return getUnifier();
  }
}

protected Operator.Unifier getUnifier()
{
   return null;
}
``
2. Do following in child class:
``
@Override
protected Operator.Unifier<Double> getUnifier()
{
    return new UnifierSumNumber<>();
}
``